pkg/store/store.go

```diff
@@ -60,3 +64,15 @@ func (s *MemoryStore) PutAcceptance(ctx context.Context, bidCtx *types.BidContex
 func (s *MemoryStore) GetValidatorRegistrations(ctx context.Context, publicKey *types.PublicKey) ([]types.SignedValidatorRegistration, error) {
 	return s.registrations[*publicKey], nil
 }
+
+func (s *MemoryStore) GetCountValidatorsRegistrations(ctx context.Context) (int, error) {
+	var totalRegistrations int
+	for _, signedRegistrations := range s.registrations {
+		totalRegistrations += len(signedRegistrations)
+	}
+	return totalRegistrations, nil
+}
+
+func (s *MemoryStore) GetCountValidators(ctx context.Context) (int, error) {
+	return len(s.registrations), nil
+}
```

Comment:
minor but the API is a bit safer if we return `uint` as we know these are values greater than or equal to 0